src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -89,19 +86,116 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
         addHighlightSet: true,
       });
       closeStepperModal();
+      const handlePublishTrackEvent = () => {
+        const trackInfo = {
+          stepper_modal: {
+            current_step: currentStep,
+            current_step_position: steps.indexOf(currentStep) + 1,
+          },
+          created_highligh_response_result_data: {
+            card_image_url: transformedHighlightSet.cardImageUrl,
+            is_published: transformedHighlightSet.isPublished,
+            highlight_set_title: transformedHighlightSet.title,
+            highlight_set_uuid: transformedHighlightSet.uuid,
+            highlighted_content_uuids: transformedHighlightSet.highlightedContentUuids,
+          },
```

Comment:
I don't think it's hugely beneficial to include all this metadata about the newly published highlight set, when what we care most about here for the event would be the newly published highlight set's UUID, which can be used at time of analysis to get at all this metadata anyways.